src/cartesian/ErrorBar.tsx

```diff
@@ -130,5 +162,9 @@ ErrorBar.defaultProps = {
   width: 5,
   offset: 0,
   layout: 'horizontal',
+  isAnimationActive: true,
```

Comment:
this is a breaking change (but this is the 3.x branch so that's ok)

Reply:
Why is it breaking? New feature, enabled by default, nothing breaks.

Reply:
We will have it in 3.x release anyway so ... it doesn't really matter. But I don't see it as breaking.

Reply:
By breaking I mean it changed default behavior. If someone expects error bars to animate but on a patch release now they do, then I would consider that "breaking" in the sense that it should be defaulted to false until the next major release. In this case it doesn't matter though